### DISCUSSION TITLE
Displaying live results in Wireshark during OMNet++ simulation

### DISCUSSION THREAD

**Discussion by Sa7bouch** [omnetpp/omnetpp#1027] [Q&A] [answered]
Hi  everyone 😊,

It is possible directly displaying **live results** in Wireshark **during** the simulation in OMNeT++ (GUI) ?
If yes how can I achieve it ?

Many thanks in advance!

**Comment by rhornig** [comment, marked as answer]
Not inside the OMNeT++ GUI (however the packet trace in the log window gives a somewht similar functionality). You may try to install PCAP writers in the simulation and open those files in wireshark, but I'm not sure whether wireshark can correctly display a file that is currently being written by an other process.
